eng/scripts/Install-GitHooks.ps1

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env pwsh
```

Comment:
I highly recommend against using git hooks as they generally do nothing but cause issues. I'm fine if folks want to play with them but we should not require or depend on them because we often have to disable them.